lib/src/ui/movies_page.dart

```diff
@@ -118,6 +123,7 @@ class _MoviesPageState extends State<MoviesPage> {
             children: <Widget>[
               CardSwiper(
                 trendingMovies: _trendingMovies,
+                bloc: _moviesBloc,
```

Comment:
Is possible to send only the boolean instead of the complete bloc?. And please rename the parameter name is not descriptive.

Reply:
done.